script/Arcueid/Day 7/07_00_ARC07_3E.txt

```diff
@@ -244,11 +242,13 @@ I remove my glasses.
 [sha:8140963fa807ccac93e0956153ffb69ba58cac36]{
 -- Page 290, Offset 17285.
 -- こめかみには軽い頭痛。
+<<<<<<< Updated upstream
 A light headache pulses in my temple.
+=======
```

Comment:
:git_is_hard:

Reply:
Ah is this anything I need to fix up in my text editor? If so please let know how